lib/admin/components/ServerSettings.js

```diff
@@ -218,56 +217,60 @@ class ServerSettings extends Component<Props, State> {
                 </h4>
               </Col>
             </Row>
-          }>
-          <div>
-            {otpServers && otpServers.length
-              ? otpServers.map((server, index) => (
-                <CollapsiblePanel
-                  data={server}
-                  defaultExpanded={!server.name}
-                  fields={SERVER_FIELDS}
-                  index={index}
-                  key={`server-${index}`}
-                  onChange={this._getOnChange}
-                  onEnter={this._onExpandServer}
-                  onRemove={this._onDeleteServer}
-                  onSave={this._onSaveServer}
-                  saveDisabled={!this.state.hasEdits}
-                  showButtonsOnBottom
-                  testId={server.name ||
-                    `[${this.messages('deployment.otpServers.serverPlaceholder')}]`}
-                  title={server.name
-                    ? <span>
-                      {server.name}{'  '}
-                      <small>{server.publicUrl}</small>
-                    </span>
-                    : `[${this.messages('deployment.otpServers.serverPlaceholder')}]`
-                  }
-                >
-                  <ServerSpecialFields
+          </Panel.Heading>
+          <Panel.Body>
+            <div>
+              {otpServers && otpServers.length
+                ? otpServers.map((server, index) => (
+                  <CollapsiblePanel
+                    data={server}
+                    defaultExpanded={!server.name}
+                    fields={SERVER_FIELDS}
                     index={index}
-                    getOnChange={this._getOnChange}
-                    server={server}
-                    terminateEC2InstanceForDeployment={this.props.terminateEC2InstanceForDeployment}
-                    terminateEC2Instances={terminateEC2Instances}
-                    onChangeProject={this._onChangeProject}
-                    onToggleEc2Info={this._onToggleEc2Info}
-                    projects={projects} />
-                </CollapsiblePanel>
-              ))
-              : <p className='lead text-center'>No servers defined</p>
-            }
-          </div>
+                    key={`server-${index}`}
```

Comment:
Use the server id so that after a server is deleted, all entries appear collapsed.
```suggestion
                    key={server.id}
```

Reply:
Good idea thank you! Refactored in 415f56f